sub-packages/bionemo-geneformer/tests/bionemo/geneformer/scripts/test_pydantic_train.py

```diff
@@ -131,9 +202,8 @@ def test_pretrain_cli(tmpdir, data_path):
 @pytest.mark.slow
 def test_finetune_cli(tmpdir, data_path):
     """Uses CLI to invoke the entrypoint"""
-    # data_path: Path = load("single_cell/testdata-20240506") / "cellxgene_2023-12-15_small" / "processed_data"
     result_dir = Path(tmpdir.mkdir("results"))
-    checkpoint_path: Path = load("geneformer/10M_240530:2.0")
+    checkpoint_path: Path = load("geneformer/10M_240530:2.0", source="pbss")
```

Comment:
Remove the `source="pbss"` please. An external user can't run this if it's there. If you want to use PBSS globally in your tests (or in CI) we can do that with the environment variable: `DEFAULT_SOURCE: SourceOptions = os.environ.get("BIONEMO_DATA_SOURCE", "ngc")  # type: ignore`. You can `BIONEMO_DATA_SOURCE=pbss py.test ...`